test/components/submission/feed-entry.spec.js

```diff
@@ -176,9 +176,14 @@ describe('SubmissionFeedEntry', () => {
       mountComponent().first('.body').html().should.equal('<div class="body"><p><b>bold</b></p>\n</div>');
     });
 
-    it('removes script tags and sanitizes html', () => {
-      testData.extendedComments.createPast(1, { body: '<script>foo</script>' });
+    it('removes script and svg tags and sanitizes html', () => {
+      testData.extendedComments.createPast(1, { body: '<script>foo</script><svg>bar</svg>' });
       mountComponent().first('.body').html().should.equal('<div class="body"></div>');
     });
+
+    it('removes style and class attributes', () => {
+      testData.extendedComments.createPast(1, { body: '<b style="color: red;" class="c">foo</b>' });
```

Comment:
Ah somehow this comment wasn't attached to my previous review. 😅

Could we also test a data attribute here? (Those are actually the attributes I worry the most about.) Could be a fake data attribute like `data-foo="bar"`.

Reply:
Done!